### PR TITLE
New service scenariorunresults

### PR DESCRIPTION
**Pull request by Leopold-Cramer**

### CODE REVIEW

File: scenariorunresult/src/main/openapi/scenariorunresult.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      description:
+      value:
+        id: Id
+        results:
+          probe1:
+            var1: var
+            value1: value
+          probe2:
+            var1: var
+            value1: value
```
Comment (jreynard-code) [inline scenariorunresult/src/main/openapi/scenariorunresult.yaml:124]: Maybe add more realistic data

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM :smile: 
Just some remarks ;)

**Review by jreynard-code — CHANGES_REQUESTED** [review]
Just some remarks

**Comment by lalepee** [comment]
image

**Review by jreynard-code — APPROVED** [review]
LGTM

**Pull request merged**